<ui version="4.0" >
 <class>Form3</class>
 <widget class="QWidget" name="Form3" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>200</width>
    <height>115</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>200</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="enabledB" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Włącz wtyczkę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="drvlistB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Urządzenie:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>drvlistB</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
